Kontent.Ai.Delivery.Caching/CacheHelpers.cs

```diff
@@ -47,7 +47,7 @@ public static class CacheHelpers
         /// <returns>Dependency key</returns>
         public static string GetItemKey<T>(string codename, IEnumerable<IQueryParameter> parameters)
         {
-            return StringHelpers.Join(new[] { typeof(T).GetHashCode().ToString(), CONTENT_ITEM_TYPED_IDENTIFIER, codename }.Concat(parameters?.Select(x => x.GetQueryStringParameter()) ?? Enumerable.Empty<string>()));
+            return StringHelpers.Join(new[] { typeof(T).FullName.ToLower(), CONTENT_ITEM_TYPED_IDENTIFIER, codename }.Concat(parameters?.Select(x => x.GetQueryStringParameter()) ?? Enumerable.Empty<string>()));
```

Comment:
I'd lose the `ToLower()`, C# is a case sensitive language.

Reply:
You are right. I fixed it and updated unit tests